tests/Doc/HomeJsonExample.cs

```diff
@@ -0,0 +1,176 @@
+// EXAMPLE: cs_home_json
+
+// STEP_START import
+using NRedisStack.RedisStackCommands;
+using NRedisStack.Search;
+using NRedisStack.Search.Aggregation;
+using NRedisStack.Search.Literals.Enums;
+using NRedisStack.Tests;
```

Comment:
would move this into 'HIDE' region.
at first sight i was thinking this should go into 'REMOVE' region, but then i realized
we left some test project related stuff in 'HIDE' regions overall the docs, is it on purpose ?